content/verein/constitution.fr.md

```diff
@@ -0,0 +1,181 @@
+{
+    "title": "Constitution",
+    "slug": "constitution",
+    "aliases": ["statutes", "statute"],
+    "type": "page"
+}
+
+Notre constitution fixe le cadre juridique de l'association Datenanfragen.de e.&nbsp;V. Elle explique entre autres nos objectifs et la manière dont nous voulons les atteindre, elle fixe les règles concernant l'adhésion à notre association et définit le processus décisionnel.
+
+<p class="box box-info" style="margin-bottom: 20px;">
+Comme nous sommes basés en Allemagne, seule la <a href="https://www.datenanfragen.de/verein/satzung">version allemande</a> de notre constitution est juridiquement applicable. Cette traduction n'est proposée que par souci de confort.<br>
+Dans les cas où nous pensons que la traduction française peut être ambiguë, nous avons fait référence au texte original allemand [entre crochets] pour plus de clarté.
```

Comment:
Seems like it is easier to convey German legal concepts in French than it is in English. :D As you didn't make use of any clarifications like this, we can remove this sentence.

Reply:
You're right. I forgot to remove it. When I was not sure I checked the german version too.
I still make my translations with the help of DeepL and it seems that it is quite good at translating "legal" sentence style. Because it's based on previously translated texts and AI it's really accurate.